pkg/action/upgrade_test.go

```diff
@@ -18,7 +18,11 @@ package action
 
 import (
 	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
 	"testing"
+	gotime "time"
```

Comment:
generally speaking standard libraries are referenced by their usual name, so `time` should not be referenced as `gotime` - kinda redundant when the code's already written in Go anyways.

If there's a third-party library attempting to munging with a similar namespace as a stdlib, then we provide an alias for the third-party library, but the standard lib remains the same.

For example `time` and `helmtime` as referenced here: https://github.com/helm/helm/blob/21251378b432fb629cb1a49af3892e20c1ec9624/pkg/action/hooks.go#L21-L26

Reply:
Thanks for pointing out this, I already change all the entries I introduced